disperser/batcher/grpc/dispatcher.go

```diff
@@ -47,13 +47,27 @@ func (c *dispatcher) sendAllChunks(ctx context.Context, state *core.IndexedOpera
 	for id, op := range state.IndexedOperators {
 		go func(op core.IndexedOperatorInfo, id core.OperatorID) {
 			blobMessages := make([]*core.BlobMessage, 0)
+			hasAnyBundles := false
 			for _, blob := range blobs {
+				if _, ok := blob.BundlesByOperator[id]; ok {
+					hasAnyBundles = true
+				}
 				blobMessages = append(blobMessages, &core.BlobMessage{
 					BlobHeader: blob.BlobHeader,
 					// Bundles will be empty if the operator is not in the quorums blob is dispersed on
 					Bundles: blob.BundlesByOperator[id],
 				})
 			}
+			if !hasAnyBundles {
```

Comment:
when would this happen, or it is just a sanity check

Reply:
Just a santiy check.